solana/Makefile

```diff
@@ -0,0 +1,33 @@
+.DEFAULT_GOAL = build
+.PHONY: build test unit-test integration-test clean
+
+test: unit-test integration-test
+
+unit-test:
+	cargo clippy --features devnet -- --allow clippy::result_large_err
+	cargo test --features devnet
+	
+integration-test:
+	anchor test --arch sbf
+
+build:
+	@echo "> Building programs"
+	anchor build --arch sbf -- --features $(NETWORK) -- --no-default-features
+
+clean:
+	rm -rf node_modules .anchor
+	anchor clean
+
+#phony targets above, non-phony targets below
+
+ifneq ($(NETWORK), $(shell cat target/network_changed 2>/dev/null))
+#force update of target/network_changed if value of NETWORK changed from last time
+.PHONY: target/network_changed
+endif
+target/network_changed:
```

Comment:
this target doesn’t seem to be used for anything, the `build` target is phony anyway

Reply:
Fixed